packages/example-app-base/src/components/SideRail/SideRail.styles.ts

```diff
@@ -76,7 +76,7 @@ export const getStyles: IStyleFunction<ISideRailStyleProps, ISideRailStyles> = p
     },
     jumpLinkSection: {
       selectors: {
-        [`@media screen and (max-width: ${uhfScreenMaxLg}px)`]: {
+        '@media screen and (max-width: 1360px)': {
```

Comment:
Do we not have a constant for this?

Reply:
We don't, it's only used as a breakpoint for the SideRail.